app/app.go

```diff
@@ -248,14 +252,15 @@ type App struct {
 	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
 	ScopedWasmKeeper     capabilitykeeper.ScopedKeeper
 
-	aolKeeper   aolkeeper.Keeper
-	didKeeper   didkeeper.Keeper
-	burnKeeper  burnkeeper.Keeper
-	tokenKeeper tokenkeeper.Keeper
-	wasmKeeper  wasm.Keeper
+	aolKeeper    aolkeeper.Keeper
+	didKeeper    didkeeper.Keeper
+	burnKeeper   burnkeeper.Keeper
+	tokenKeeper  tokenkeeper.Keeper
+	wasmKeeper   wasm.Keeper
+	MarketKeeper marketkeeper.Keeper
```

Comment:
```suggestion
	marketKeeper marketkeeper.Keeper
```
다른 keeper와 마찬가지가지로 소문자로 가면 어떨까요